pkg/cmd/cli/describe/printer.go

```diff
@@ -36,7 +36,7 @@ var (
 	projectColumns          = []string{"NAME", "DISPLAY NAME", "STATUS"}
 	routeColumns            = []string{"NAME", "HOST/PORT", "PATH", "SERVICE", "LABELS", "INSECURE POLICY", "TLS TERMINATION"}
 	deploymentColumns       = []string{"NAME", "STATUS", "CAUSE"}
-	deploymentConfigColumns = []string{"NAME", "TRIGGERS", "LATEST"}
+	deploymentConfigColumns = []string{"NAME", "REVISION", "REPLICAS", "TRIGGERED BY"}
```

Comment:
Also not showing replicas so far has been really disturbing. We need though to show current alongside desired, showing only desired is misleading. We also need to make use of -o wide and put images, containers, and the selector under it. Preparing a pull for that.